Core.Collectors/Web/HttpExceptionSignature.cs

```diff
@@ -8,22 +8,37 @@ namespace Microsoft.CloudMine.Core.Collectors.Web
 {
     public class HttpExceptionSignature
     {
-        public static readonly HttpExceptionSignature RequestTimeoutException = new HttpExceptionSignature(exception => 
-        {
-            Type exceptionType = exception.GetType();
-            return exceptionType == typeof(TaskCanceledException) && exception.Message.Equals("The operation was canceled.");
-        });
-
         private readonly Func<Exception, bool> matcher;
+        private readonly Func<Task> handler;
```

Comment:
I thought that this would be a list of CollectionNodes (to be executed instead of the original collection node). Did we decide to change how we are doing things?